src/shared/card-hover.js

```diff
@@ -145,6 +145,8 @@ export function attachDraftRatings(card, ratingsContainer) {
   let rankValues = card.rank_values;
   let rankControversy = card.rank_controversy;
 
+  // TODO: Refactor and enable this rule.
+  // eslint-disable-next-line prefer-spread
   let maxValue = Math.max.apply(Math, card.rank_values);
```

Comment:
This is the only occurrence of this rule violation, and should be done first.

Reply:
The rule is asking for this edit:
```suggestion
  let maxValue = Math.max(...rank_values);
```
It runs more code, but the rule is trying to establish spread as the preferred idiom.